basics/dataproc_cluster/stable/variables.tf

```diff
@@ -0,0 +1,22 @@
+## --------------------------------------------------------------
+## Mandatory variable definitions
+## --------------------------------------------------------------
+
+variable "gcp_project_id" {
+  type        = string
+  description = "The GCP project ID to create resources in."
+}
+
+# Default value passed in
+variable "gcp_region" {
+  type        = string
+  description = "Region to create resources in."
+  default     = "us-central1"
```

Comment:
This should not have a default set - the value should be provided by the platform.